### ISSUE TITLE
Accessibility improvements with screen reader NVDA

### ISSUE THREAD

**Issue by shadow2560** [Rem0o/FanControl.Releases#4025]
Accessibility of the GUI is realy partial.

I use NVDA as a screen reader but some functions are totally not accessible. With object navigation of the screen reader I can access some objects or some infos (tabs labels, lists items) and with keyboard navigation I can modify some.

But some items are not labeled (those like the button that can be used to set the steps for a specific fan, the only button like that labeled correctly is the one that activate the fan control to set a curve in the list). Also, the button to add a curve, nor by navigating with the keyboard neither with the objects navigation of the screen reader (maybe adding a keyboard shortcut could be a good first solution) (maybe there is a button to delete a curve but I don't see it).

Also, adding a shortcuts list in the readme (and maybe in a text file in the releases files) could be a good idea cause the only one described by the screen reader in the GUI is the "ctrl+r" shortcut to refresh detection but I find others like "ctrl+s" to save the config, "ctrl+l" to load a config (probably cause not described) and "ctrl+n" to create a new config file.

For now that's all I can describe, maybe there is others accessibility problems but without access to all elements it's difficult to describe correctly those problems.

Thanks for this work, having an accessible program to set fans with Windows could be a real improvement cause all others programs that I have tested to do that are far worst than your, in general they are totally not accessible but your is partially accessible and I think it doesn't needs lots of improvement to be at least usable.

**Comment by Rem0o** [comment]
Dupe of https://github.com/Rem0o/FanControl.Releases/issues/3841
